RakutenShop/Scenes/RecipesView/RecipesViewController.swift

```diff
@@ -53,19 +57,19 @@ class RecipesViewController: BaseViewController {
             }
         }
         
+        viewModel.errorMessage.producer.skip(first: 1).startWithValues { (errorMessage) in
+            DispatchQueue.main.async {
+                SVProgressHUD.showError(withStatus: errorMessage)
+            }
+        }
+        
         viewModel.recipeItems.producer.skip(first: 1).startWithValues { (recipes) in
```

Comment:
maybe start function instead of startWithValues will be better, just try I am not sure.